app/views/home/index.html.erb

```diff
@@ -1,3 +1,113 @@
-<div class="container-fluid" ng-app="receta">
-A-Pc
-</div>
+<!-- Navigation -->
+<nav class="navbar navbar-default" role="navigation">
```

Comment:
We will have an admin panel. Would be nice to be able to import the header and navigation code instead of writing it twice.